mesop/components/table/e2e/table_app.py

```diff
@@ -57,5 +57,5 @@ def on_click(e: me.TableClickEvent):
   state = me.state(State)
   state.selected_cell = (
     f"Selected cell at col {e.col_index} and row {e.row_index} "
-    f"with value {str(df.iat[e.row_index, e.col_index])}"
+    f"with value {df.iat[e.row_index, e.col_index]!s}"
```

Comment:
Nice. I didn't know about the `!s` syntax in the formatted string

Reply:
me neither!